addon/components/lt-body.js

```diff
@@ -163,6 +163,21 @@ export default Component.extend({
   overwrite: false,
 
   /**
+   * If true, the body will prepend an invisible `<tr>` that scaffolds the
+   * widths of the table cells.
+   *
+   * ember-light-table uses [`table-layout: fixed`](https://developer.mozilla.org/en-US/docs/Web/CSS/table-layout).
+   * This means, that the widths of the columns are defined be the first row
+   * only. By prepending this scaffolding row, widths of columns only need to
+   * be specified once.
+   *
+   * @property enableScaffolding
+   * @type {Boolean}
+   * @default true
+   */
+  enableScaffolding: true,
```

Comment:
Maybe it would be more sensible to make this a CP that's the inverse of [`overwrite`](https://github.com/buschtoens/ember-light-table/blob/fa85616464e1fdbd6b27d1c2e9ef8746d1f833dc/addon/components/lt-body.js#L155-L163).

As we're prepending a scaffolding row, this could lead to some unexpected behavior for people that use selectors like `:first-child`.